scripts/spec.py

```diff
@@ -0,0 +1,52 @@
+from datetime import date
+import json
+
+import bokeh.models as bm; bm
+import bokeh.models.widgets as bmw; bmw
```

Comment:
@mattpap there's plenty to aspire to in terms of quality. Until we have hundreds of QA people like BMW certainly has, I'm sure that will always be the case. In the mean time we've managed to create a tool that is evidently useful to a fairly large number of people, with only a handful of full-time contributors.